packages/zoe/src/zoeService/instanceAdminStorage.js

```diff
@@ -24,7 +24,7 @@ const InstanceAdminStorageShape = M.interface('InstanceAdminStorage', {
   getOfferFilter: M.call(M.eref(InstanceHandleShape)).returns(M.promise()),
   setOfferFilter: M.call(M.eref(InstanceHandleShape)).returns(),
   getInstallationForInstance: M.call(InstanceHandleShape).returns(M.promise()),
-  getInstanceAdmin: M.call(InstanceHandleShape).returns(InstanceAdminShape),
+  getInstanceAdmin: M.call(InstanceHandleShape).returns(InstanceAdminI),
```

Comment:
```suggestion
  getInstanceAdmin: M.call(InstanceHandleShape).returns(InstanceAdminShape),
```

With the exception of `M.await`, the parameter and result elements of method guards must be patterns, not guards. A value is checked against a pattern. An implementation implements an interface.

Reply:
done